1223-curator_worklist/my_worklist_approval.feature

```diff
@@ -25,15 +25,22 @@ Feature: Curator handles an Approval Request
 	@3188
 	Scenario: Curator approves an Approval Request to publish
 		Given the User wants to publish a Resource
-		And the Editor has restricted publishing rights to Curators
+		And Institutions publications policy is "Only Curator can publish" 
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the Resource is published
 
+	@3191
+	Scenario: Curator declines an Approval Request
+		When the Curator declines the Request 
+		And Institutions publications policy is "Only Curator can publish" 
```

Comment:
Institutional or Institution's (I suspect the former…it makes more sense, otherwise you'd need something about affiliation)